### PR TITLE
Changing default range for stock chart behavior to show all data.

### PR DESCRIPTION
**Pull request by anutron**
@davywentworth 

Confirmed this works with any amount of data. The "All" is a special button that just selects everything, regardless of how much you have (docs: http://api.highcharts.com/highstock#rangeSelector.buttons).

### CODE REVIEW

File: dist/dev/js/behavior-ui.js [hunk 1]
```diff
@@ -18585,7 +18585,9 @@ Chart.Stock = new Class({
   options: {
     // the color of the series in the navigator; hex, rgb, or rgba
     // if not specified uses the first color in the colors array in options
-    navigatorSeriesColor: null
+    navigatorSeriesColor: null,
+    // default selected range = 1/mo
+    selectedRange: 0
```
Comment (davywentworth) [inline dist/dev/js/behavior-ui.js]: shouldn't this also be 6?

**Comment by davywentworth** [comment]
This LGTM, but does it do what the ticket asked for? Wasn't it supposed to show just a year's worth by default?

File: dist/dev/js/behavior-ui.js (hunk 1 above)
Comment (anutron) [inline dist/dev/js/behavior-ui.js]: I left the class itself alone and just changed the behavior.

**Comment by anutron** [comment]
Would have prefered a year, but using 1Y has that exception problem and "all" doesn't, so went w/ that.

**Pull request closed without merging**